### ISSUE TITLE
Focus issues with Popover

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#3067]
# Info

If you create a Popover around an interactive element (button for example). Follow these steps:

1. Click interactive element to open the Popover
2. Close the popover by hitting the Escape button
3. Notice nothing is focused
4. Hit tab, and notice the popover element is then re-focused.

## Acceptance Criteria:

Here's what should happen:

1. Click interactive element to open the Popover
2. Close the popover by hitting the Escape button
3. Interactive element should still be focused
4. Hit tab, and the next interactive element should be selected

**Comment by twjeffery** [comment]
is this a related issue? https://github.com/GovAlta/ui-components/issues/2773
